tests/system_tests_tcp_conns_terminate.py

```diff
@@ -0,0 +1,387 @@
+#
```

Comment:
@gabordozsa can you please add this test to the list of tests in tests/CMakeLists.txt ? This test is not currently executing in CI

Reply:
I added it to the list.